src/app/clusters/ias-zone-server/ias-zone-server.mak

```diff
@@ -0,0 +1,42 @@
+# ias-zone-server.mak
```

Comment:
Do we plan to use this makefile?

Reply:
not looking carefully at what got pulled in from my cherry-picking, will delete